### DISCUSSION TITLE
Couleur des traces

### DISCUSSION THREAD

**Discussion by PierreDubois51** [mytourbook/mytourbook#1235] [Ideas]
la couleur des traces va du maxi au mini en fusionnant des couleurs. Serait-il possible d'avoir des couleurs de traces par tranches. Exemple pour les pentes 0 à 2% blanc;2 à 4% bleu;4 à 6% ect... je pense que cela serait plus lisible 
Cordialement

**Comment by FJBDev** [comment]
Pierre,

Pouvez-vous creer une "Feature Request" pour cette fonctionalite ([lien](https://github.com/mytourbook/mytourbook/issues)) ? (Si possible, de preference en anglais)
